packages/api/src/providers/fitbit.ts

```diff
@@ -347,9 +358,9 @@ export class Fitbit extends Provider implements OAuth2 {
     );
   }
 
-  async postAuth(token: string, userId?: string) {
+  async postAuth(token: string, userId: string) {
     const accessToken = JSON.parse(token).access_token;
-
+    const fitbitUserId = JSON.parse(token).user_id;
     const scopes = JSON.parse(token).scope;
```

Comment:
just store the result of `JSON.parse(token)` in a variable, so you don't have to parse the token 3 times